Files/MultilingualResources/Files.zh-Hans.xlf

```diff
@@ -2371,35 +2371,35 @@
         </trans-unit>
         <trans-unit id="NavToolbarColumnView.AutomationProperties.Name" translate="yes" xml:space="preserve">
           <source>Column View</source>
-          <target state="new">Column View</target>
+          <target state="new">列表视图</target>
```

Comment:
Please adjust the state for translated strings. See the docs for more information https://files-community.github.io/docs/#/articles/translating-files?id=working-with-resource-files.